src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1252,6 +1252,59 @@ int xcldev::device::bandwidthKernelTest(void)
     return 0;
 }
 
+int xcldev::device::hostBandwidthKernelTest(void)
+{
+    std::string output;
+
+    if ((sensor_tree::get<std::string>("system.linux", "N/A").find("Red Hat") != std::string::npos)
+            && (sensor_tree::get<std::string>("system.machine", "N/A").find("ppc64le") != std::string::npos)) {
+        std::cout << "Testcase not supported on Red Hat and PowerPC. Skipping validation"
+                  << std::endl;
+        return -EOPNOTSUPP;
+    }
```

Comment:
Since we don't depend on pyopencl, this should be removed as well. While you're at it can you remove it from bandwidthKernelTest() as well? Thanks